meeshkan/config.py

```diff
@@ -4,6 +4,7 @@
 import yaml
 import logging
 
+DEFAULT_SPECS_DIRECTORY = "specs/"
```

Comment:
My only hesitation looking at this again is, should it be `DIR` rather than `DIRECTORY`? Considering https://github.com/meeshkan/meeshkan/blob/master/meeshkan/config.py#L12-L13 🤔